stories/features/newTab/default/footerInfo.tsx

```diff
@@ -21,6 +21,13 @@ interface Props {
 }
 
 export default class FooterInfo extends React.PureComponent<Props, {}> {
+
+  onKeyPressSettings = (e: React.KeyboardEvent<HTMLButtonElement>) => {
```

Comment:
Why is this necessary? If this is a <button element then we get this for free, along with other accessibility benefits.

Reply:
This handler was added in order to add a conditional to the onKeyDown for which button was pressed. Otherwise, simple pressing "Tab" through the page element will toggle the menu. 

We wanted to implement the functionality to "click outside the menu to close it" which caused a double trigger on the button when we clicked on it. Thus, the reason I had to add an additional set of logic around `onKeyDown` was because `onClick` had to be disabled in that case.